packages/rest/src/sequence.ts

```diff
@@ -121,3 +122,83 @@ export class DefaultSequence implements SequenceHandler {
     }
   }
 }
+
+/**
+ * A sequence implementation using middleware chains
+ */
+export class MiddlewareSequence implements SequenceHandler {
+  static defaultOptions: InvokeMiddlewareOptions = {
+    chain: RestTags.REST_MIDDLEWARE_CHAIN,
+    orderedGroups: [
```

Comment:
Why is this named `orderedGroups`? Do `sendResponse`, `cors`, `apiSpec` etc., resolve to arrays of middleware? If not, the name is misleading. If yes, the fact has to be clearly documented.

Reply:
After going through the codebase, it has become clear to me. A name like `executionOrder` or something like that might make it more obvious.

Reply:
We inherit `orderedGroups` from interceptors - middleware is a specialized interceptors that are invoked as part of the sequence.